config.dev.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Could we instead put this on the documentation?

Reply:
We'll do that as part of https://github.com/PrairieLearn/PrairieLearn/issues/11934.